code/modules/mining/machine_processing.dm

```diff
@@ -108,6 +110,8 @@
 	var/list/ores_stored
 	var/static/list/alloy_data
 	var/active = 0
+	var/speed_process = 0 // are we on SSfastprocess?
```

Comment:
```suggestion
	var/speed_process = FALSE // are we on SSfastprocess?
```
you should probably use FALSE instead of just 0, like you did for the rest.